code/edgeidentity/build.gradle

```diff
@@ -259,6 +262,7 @@ dependencies {
     testImplementation 'org.powermock:powermock-module-junit4:2.0.0'
     testImplementation 'org.json:json:20180813'
 
+    androidTestImplementation "androidx.test:core:1.4.0"
```

Comment:
For line 218 and 234,
def excludeRegex = ['**/ADB*.class', '**/BuildConfig.class']

Similar to Consent, if there is no ADB class file, we can remove that
def excludeRegex = ['**/BuildConfig.class']